operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -413,6 +409,36 @@ spec:
 # Only used when cache_enabled is true.
 #    ---
 #    cache_duration: 300000000
+#
```

Comment:
Note: all values in this kiali_cr.yaml document for the user what the actual defaults are. So users can come to this file and see what the operator's default values are in case the user does not set these values themselves.

I see some values here are not the actual defaults. Change these so it matches the defaults yaml. I'll give suggestions below.